pyccel/codegen/printing/cucode.py

```diff
@@ -61,14 +61,45 @@ def _print_Module(self, expr):
             else:
                 local_imports += self._print(imp)
 
-        imports = f'{c_headers_imports}\
-                    extern "C"{{\n\
-                    {local_imports}\
-                    }}'
+        imports = f'{c_headers_imports}\n\
+                    {local_imports}\n'
 
         code = f'{imports}\n\
                  {global_variables}\n\
                  {body}\n'
 
         self.exit_scope()
         return code
+
+    def _print_ModuleHeader(self, expr):
+        self.set_scope(expr.module.scope)
+        self._in_header = True
+        name = expr.module.name
+        if isinstance(name, AsName):
+            name = name.name
+        # TODO: Add interfaces
```

Comment:
Is this TODO necessary? Can it be fixed before merging? E.g. by raising an error to warn the users that Interfaces cannot be printed in Cuda?

Reply:
fixed d48329c

Reply:
When I read the code again, the 'ModuleHeader' object has no attribute 'interfaces', so I can't raise an error for now

Reply:
Ah. In that case it seems like this TODO is not valid. There is nothing to do

Reply:
The same  in C, so I think the best is to keep the Todo so everyone can see it and maybe someone is going to fix it in the future, I guess ??

Reply:
No. Interfaces are fully supported in C

Reply:
If they don't appear in the `ModuleHeader` it is probably because they are not necessary. C/cuda don't have an equivalent to fortran's `interface` keyword so nothing needs to be handled in the header. That is why nobody bothered to add the section to `ModuleHeader`

Reply:
I said that because I see that todo in C code, and is that normal ??

Reply:
ah oky that make sense

Reply:
Hmm that looks like an oversight that I should clean up at some point in the main repo

Reply:
eah, for sure. But what I should do now, I think removing that 'todo' is the best thing I can do

Reply:
fixed 74d7a34